fetcher/fetcher.go

```diff
@@ -56,6 +56,16 @@ type Fetcher interface {
 	GetOracleExadataCellDisks() map[agentmodel.StorageServerName][]model.OracleExadataCellDisk
 
 	// Microsoft/SQLServer fetchers
+	GetMicrosoftSQLServerInstances() []agentmodel.ListInstanceOutputModel
+	GetMicrosoftSQLServerInstanceInfo(conn string, inst *model.MicrosoftSQLServerInstance)
+	GetMicrosoftSQLServerInstanceEdition(conn string, inst *model.MicrosoftSQLServerInstance)
+	GetMicrosoftSQLServerInstanceLicensingInfo(conn string, inst *model.MicrosoftSQLServerInstance)
```

Comment:
This is a get but doesn't return any value :thinking:  why?
Maybe just call it `Run` instead of `Get`?

Reply:
Because the result of marshalling is directly putted into inst...

Reply:
:+1: